segmentstore/server/src/test/java/io/pravega/segmentstore/server/store/StreamSegmentServiceNoOpWriteOnlyTests.java

```diff
@@ -1,97 +0,0 @@
-/**
- * Copyright (c) Dell Inc., or its subsidiaries. All Rights Reserved.
```

Comment:
Can you elaborate the reason to remove these tests?

Reply:
These are end-to-end segment store integration tests, which test LTS and recovery from LTS. With No-Op mode there is no data written to LTS so any attempt to read it from there will fail. It was by pure chance that these tests passed when they did, and any attribute or cache eviction that might have happened would have caused it to fail. 

There is no point in doing this kind of test with No-Op, so I removed both of them.